server/models/schemas/user.js

```diff
@@ -110,8 +110,20 @@ const schema = new Schema({
             objectsScaling: { type: Types.String, required: false, enum: ['default', 'clamped'], default: 'default' },
             objectsMinimumScale: { type: Types.Number, required: false, default: 8 },
             objectsMaximumScale: { type: Types.Number, required: false, default: 16 },
-            nebulaDensity: { type: Types.String, required: false, enum: ['none', 'sparse', 'standard', 'abundant'], default: 'standard' },
             antiAliasing: { type: Types.String, required: false, enum: ['enabled', 'disabled'], default: 'enabled' },
+            background:{
+              nebulaFrequency: { type: Types.Number, required: false, default: 12 },
+              nebulaDensity: { type: Types.Number, required: false, default: 3 },
+              nebulaOpacity: { type: Types.Number, required: false, default: 1.0 },
+              moveNebulas: { type: Types.String, required: false, enum: ['enabled', 'disabled'], default: 'enabled' },
+              nebulaMovementSpeed: { type: Types.Number, required: false, default: 1.0 },
+              starsOpacity: { type: Types.Number, required: false, default: 1.0 },
+              blendMode: { type: Types.String, required: false, enum: ['ADD', 'NORMAL'], default: 'NORMAL' },
+              backgroundStars: { type: Types.String, required: false, enum: ['enabled', 'disabled'], default: 'enabled' },
+              nebulaColor1: { type: Types.String, required: false, default: '#ffad42' },
+              nebulaColor2: { type: Types.String, required: false, default: '#ffad42' },
```

Comment:
Is `nebulaColor2` the same as `nebulaColor1` intentionally?

Reply:
no, you can have UP TO 3 different colors, i just happened to like how it looked with 2 colors